### DISCUSSION TITLE
FastOlympicCoding Layout in SublimeText3

### DISCUSSION THREAD

**Discussion by dhimanda** [Jatana/FastOlympicCoding#36] [General]
I know this problem does not belong here but I will love to if anyone helps me to solve it.
I am using FastOlympicCoding for so long in windows. But recently I switched int MacOS. 
In windows when I set this layout it remains same. 
<img width="1440" alt="Screenshot 2021-03-15 at 4 38 19 AM" src="https://user-images.githubusercontent.com/53374747/111086856-bffb7780-8548-11eb-9529-2cb9b9272d90.png">
**but in macOS it starts from first, like** 
<img width="1440" alt="Screenshot 2021-03-13 at 10 55 30 PM" src="https://user-images.githubusercontent.com/53374747/111086876-db668280-8548-11eb-88fd-ca21a93d7ffa.png">

is there any way to fix this?

**Comment by Jatana** [comment]
Could you explain what do you mean by 'it starts from first'?

**Reply by dhimanda** [reply]
Actually, I am trying to say, SublimeText doesn't remember the layout settings, is there any way to remember this? 
Cause otherwise every time I have to set 3 columns and max 2 group settings in sublime text.

**Reply by Jatana** [reply]
Ok, I understood what you want. You can use `cmd+Q` to exit Sublime Text instead of closing the window. Then, when you reopen it, the layout should be restored.

**Reply by dhimanda** [reply]
WOW! You are just wow! Thanks, man! It just saved my life. God bless you man <3
